main.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+from database import SessionLocal
+import models
+from schemas import AuthorCreate, BookCreate
+import crud
+
+app = FastAPI()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+@app.post("/authors/", response_model=AuthorCreate)

```

Comment:
The response model for the `create_author` endpoint should be `Author` instead of `AuthorCreate` to include the `id` field in the response.